save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt

```diff
@@ -124,6 +130,28 @@ class ExecutionController(private val executionService: ExecutionService,
                     }
                 }
 
+    /**
+     * Delete all executions by project name and project owner
+     *
+     * @param name name of project
+     * @param owner owner of project
+     * @return ResponseEntity
+     * @throws ResponseStatusException
+     */
+    @PostMapping("/deleteExecution")
+    fun deleteExecutionForProject(@RequestParam name: String, @RequestParam owner: String): ResponseEntity<String>? {
```

Comment:
I think we should be able to delete executions selectively, by ID or a list of IDs. Clearing the history completely from the frontend is needed not as often.